client/src/store/ConfigureStore.js

```diff
@@ -12,6 +13,7 @@ const reducer = combineReducers({
     Jackpot,
     SignUp,
     CardRoom,
+    News,
```

Comment:
This will take some refactoring, but now that you have a good grasp on Redux / hooks, you should refactor your code so that all the slices of state are not capitalized! This will start from the import statements and the `combineReducers` in this file. Also, I think you should rename your `signUp` reducer to be a `users` reducer!

```js
import auth from './auth';
import jackpot from './jackpot';
import users from './users';
import cardRoom from './cardRoom';
import news from './news';
```

```js
const reducer = combineReducers({
    auth,
    jackpot,
    users,
    cardRoom,
    news,
});
```